myriad-scheduler/src/main/java/org/apache/myriad/Main.java

```diff
@@ -158,10 +158,8 @@ private void initProfiles(Injector injector) {
           Long cpu = Long.parseLong(profileResourceMap.get("cpu"));
           Long mem = Long.parseLong(profileResourceMap.get("mem"));
 
-          ServiceResourceProfile serviceProfile = new ExtendedResourceProfile(new NMProfile(profile.getKey(), cpu, mem),
+          ServiceResourceProfile serviceProfile = new ExtendedResourceProfile(new NMProfile(profile.getKey(), cpu, mem), taskUtils.getExecutorCpus(), taskUtils.getExecutorMemory(),
               taskUtils.getNodeManagerCpus(), taskUtils.getNodeManagerMemory());
-          serviceProfile.setExecutorCpu(taskUtils.getExecutorCpus());
-          serviceProfile.setExecutorMemory(taskUtils.getExecutorMemory());
 
```

Comment:
Putting a comment here the nodemanager/executor cpu memory limits a relic of how we used to run the executor which was as a small supervisor task.  We should clean this up.  We should verify the setting gets passed all the way though to the nodemanager as well.